licenses/licenses.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0'?>
+<licenseSummary>
+    <project>saml-android-app</project>
+    <version>0.0.0</version>
```

Comment:
@lgriffin same as the other android PR, can't find a version number in the repo

Reply:
Version number isn't hugely important as that metadata is not really used anywhere meaningful. I'd leave it at 0.0.0 and we can PR against it if we do come up with a better number.

Reply:
Awesome, just wanted to double check, feel free to merge if you are happy